### PR TITLE
Make header

### PR DESCRIPTION
**Pull request by Edjhernandez**
## Here we make the follow items:

- Top bar with logo and menu.
- Head line with personal description.

Screenshot 2023-03-09 155158

Let's work!! 🔢

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -5,17 +5,17 @@ body {
...
+  --mark-text1: #a35709;
+  --mark-text2: #ff8303;
+  --text: #f0e3ca;
 }
 
 header {
-    background-color: var(--dark-bg);
-    height: 622px;
+  background-color: var(--dark-bg);
+  height: 622px;
```
Comment (Albinpirela) [inline styles.css]: Great job @Edjhernandez 
 I can notice the quality of your code. Please 
consider changing the metric unit from (px) to (%) to have a responsive result

File: styles.css [hunk 2]
```diff
@@ -26,16 +26,16 @@ div.topbar {
 }
```
Comment (Albinpirela) [inline styles.css:26]: This selector is basic I suggest that you remove it so that it is cleaner and save lines

File: styles.css [hunk 3]
```diff
@@ -26,16 +26,16 @@ div.topbar {
...
-    display: flex;
+  display: flex;
 }
 
 div.logo p {
-  font-family: 'Impact','Times New Roman', Times, serif;
+  font-family: 'Impact', 'Times New Roman', Times, serif;
   font-style: normal;
   font-weight: 400;
   font-size: 32px;
```
Comment (Albinpirela) [inline styles.css]: I suggest changing the unit of measure from px to rem 👍

**Review by Albinpirela — APPROVED** [review]

**Review by Albinpirela — CHANGES_REQUESTED** [review]
great job @Edjhernandez 
I can the notice the quality of your code, Please consider changing 
the metric unit  and remove empty selectors

File: styles.css (hunk 1 above)
Comment (Edjhernandez) [reply styles.css]: Hi @Albinpirela, thanks.
I'll change these units px per %.
I agree with you.

File: styles.css (hunk 2 above)
Comment (Edjhernandez) [reply styles.css:26]: Thanks! @Albinpirela i am going to delete that class.

**Review by Albinpirela — CHANGES_REQUESTED** [review]
Great job @Edjhernandez 👍  . I suggest you change the units of measurement in font-size in the line 36, 50, 58, 77, and 85 🐶

**Review by Albinpirela — APPROVED** [review]
great job @Edjhernandez , the pr is approved you can merge 👍

**Pull request merged**